config/config.go

```diff
@@ -130,7 +130,8 @@ type GrafanaVariablesConfig struct {
 // TracingConfig describes configuration used for tracing links
 type TracingConfig struct {
 	Auth              Auth   `yaml:"auth"`
-	Enabled           bool   `yaml:"enabled"` // Enable autodiscover and Jaeger in Kiali
+	Enabled           bool   `yaml:"enabled"` // Enable Jaeger in Kiali
+	Integration       bool   `yaml:",omitempty"`
```

Comment:
`yaml:",omitempty"` > "`yaml:"integration,omitempty"` ?

Note that config changes should be propagated to the CRs

Reply:
This is a prop for use internally with kiali, I think that we shouldn't add to the CRs

Reply:
if so maybe that shouldn't be in config, but just some global variable?

Reply:
I moved it in the last commit. Thanks !!!